samples/extensions/vertex_dynamic_state/vertex_dynamic_state.cpp

```diff
@@ -549,7 +540,7 @@ void vertex_dynamic_state::model_data_creation()
 	    VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT | VK_MEMORY_PROPERTY_HOST_COHERENT_BIT,
 	    vertex_buffer_size,
 	    &vertex_staging.memory,
-	    vertices);
+	    &vertices);
```

Comment:
When using `std::array`, you should get the pointer to the vertices via `vertices.data()`.
And likewise for the indices.